packages/concerto-core/lib/decoratormanager.js

```diff
@@ -242,16 +297,88 @@ class DecoratorManager {
                 });
             }
         }
+    }
+
+    /**
+     * Adds decorator commands with index to the computed list
+     * @param {*} targetMap the target map to add the command to
+     * @param {targetKey} targetKey the target key to add the command to
+     * @param {DcsIndexWrapper[]} computedList the computed list to add the command to
+     * @private
+     */
+    static addToComputedList(targetMap, targetKey, computedList) {
+        const targetCommands = targetMap.get(targetKey);
+        if (targetCommands) {
+            targetCommands.forEach((dcsWithIndex) => {
+                computedList.push(dcsWithIndex);
+            });
```

Comment:
Can you push multiple items in one operation?

Reply:
Done